### PR TITLE
Added notifications about import issues

### PR DESCRIPTION
**Pull request by alex-voodoo**
This PR introduces the email notifications from the people import.

### CODE REVIEW

File: people/management/commands/import.py [hunk 1]
```diff
@@ -23,13 +27,45 @@ class Command(BaseCommand):
...
+            self.stderr.write(self.style.SUCCESS("There were no issues.  Will not notify anyone."))
+            return
+
+        report = render_to_string('people/mail_import_issues.txt', {
+            'issues': issues,
+            'persons_to_delete': persons_to_delete
+        })
+
+        message = email.message.EmailMessage()
+        message['Subject'] = 'There were issues while importing people from the external data source'
```
Comment (anarute) [inline people/management/commands/import.py:46]: Nit:

```suggestion
        message['Subject'] = '[Team App] There were issues while importing people from the external data source'
```

File: people/management/commands/import.py [hunk 2]
```diff
@@ -39,6 +75,7 @@ def handle(self, *args, **options):
                 self.stdout.write(self.style.SUCCESS("Creating {person}.".format(person=login_str)))
             else:
                 self.stdout.write(self.style.SUCCESS("Updating {person}.".format(person=login_str)))
+                persons_to_delete.remove(person)
```
Comment (anarute) [inline people/management/commands/import.py:78]: Is this change related to the notification or is it another fix? if the latter I would split it in another commit

File: people/templates/people/mail_import_issues.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{% if issues %}Issues with existing records
+----------------------------
+
+There were issues while importing:
+{% for issue in issues %}- {{ issue }}
+{% endfor %}{% endif %}
+{% if persons_to_delete %}Personal records to delete
+--------------------------
+
+Some records do not present in the data source, probably they should be removed:
```
Comment (anarute) [inline people/templates/people/mail_import_issues.txt:12]: Nit:

```suggestion
Some records are not present in the data source, probably they should be removed:
```

**Review by anarute — APPROVED** [review]
Added some nits but it looks good to me, thanks!

File: people/management/commands/import.py (hunk 2 above)
Comment (alex-voodoo) [reply people/management/commands/import.py:78]: No, it belongs to the logic.  The `persons_to_delete` initially contains a set of all records existing in the local DB, and when we process an incoming person that already existed, we remove that record from the set.  What remains in the end are candidates for removal.

File: people/management/commands/import.py (hunk 1 above)
Comment (alex-voodoo) [reply people/management/commands/import.py:46]: I think this is not needed.
The mail is sent from the domain of the service, so it can be easily filtered by the from: address.

Comment (anarute) [reply people/management/commands/import.py:46]: Ok, not a strong preference. I found it too generic and easy to lose in a mailbox, but that's personal.
